Extension/src/LanguageServer/configurations.ts

```diff
@@ -227,6 +227,7 @@ export class CppProperties {
         }
 
         const settingsPath: string = path.join(this.configFolder, this.configurationGlobPattern);
+        let isCurrentWindow: boolean = false;
```

Comment:
You'll want to put this in the CppProperties class, rather than as a local variable of the CompilerDefaults setter.  (Which begs the question, Why are we setting up file watchers, etc., in a setter for CompilerDefaults ?  But no need to fix that here...)